invoke.example.yml

```diff
@@ -1,11 +1,12 @@
 ---
 nautobot_secrets_providers:
-  project_name: "nautobot_secrets_providers"
-  nautobot_ver: "1.4.10"
+  project_name: "nautobot-secrets-providers"
+  nautobot_ver: "latest"
```

Comment:
This doesn't match what's in the `tasks.py`

Reply:
Will fix the template, keeping it here for now.

Reply:
https://github.com/networktocode-llc/cookiecutter-ntc/pull/368/commits/063a7ffc337c8dc552e5adbce53027203ea7f79d